test/e2e/adminactions_kubernetesobjects.go

```diff
@@ -0,0 +1,123 @@
+)
+
+var _ = Describe("Admin actions", func() {
+	BeforeEach(runAdminTestsInDevOnly)
+
+	Specify("kubernetesobjects list", func() {
+		result, err := adminRequest("GET", "kubernetesobjects", "", nil, "kind=configmap", "namespace=openshift-machine-api")
+		Expect(err).NotTo(HaveOccurred())
+		obj := &unstructured.Unstructured{}
+		err = obj.UnmarshalJSON(result)
+		Expect(err).NotTo(HaveOccurred())
+
+		objs, err := obj.ToList()
+		Expect(err).NotTo(HaveOccurred())
+
+		var validNames = []string{
+			"cluster-autoscaler-operator-ca",
+			"cluster-autoscaler-operator-leader",
+			"kube-rbac-proxy",
+			"kube-rbac-proxy-cluster-autoscaler-operator",
+			"machine-api-operator",
+			"machine-api-operator-images",
+		}
+
+		// get objects with kubeclient and compare
+		cm, err := Clients.Kubernetes.CoreV1().ConfigMaps("openshift-machine-api").Get("cluster-autoscaler-operator-leader", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		for _, o := range objs.Items {
+			Expect(o.GetNamespace()).To(BeEquivalentTo(cm.Namespace))
+			Expect(o.GetName()).To(BeElementOf(validNames))
+		}
+	})
+
+	Specify("kubernetesobjects get", func() {
+		// Get objects via kubernetesobjects
+		result, err := adminRequest("GET", "kubernetesobjects", "", nil, "kind=configmap", "namespace=openshift-machine-api", "name=cluster-autoscaler-operator-leader")
+		Expect(err).NotTo(HaveOccurred())
+		obj := &unstructured.Unstructured{}
+		err = obj.UnmarshalJSON(result)
+		Expect(err).NotTo(HaveOccurred())
+
+		// Confirm via kubeclient
+		cm, err := Clients.Kubernetes.CoreV1().ConfigMaps("openshift-machine-api").Get("cluster-autoscaler-operator-leader", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		Expect(obj.GetNamespace()).To(BeEquivalentTo(cm.Namespace))
+		Expect(obj.GetName()).To(BeEquivalentTo(cm.Name))
+		Expect(obj.GetAnnotations()).To(BeEquivalentTo(cm.Annotations))
+	})
+
+	Specify("kubernetesobjects create/delete", func() {
+		// Create new object and confirm via kubeclient
+		_, err := adminRequest("POST", "kubernetesobjects", `{
+			"kind": "ConfigMap",
+			"apiVersion": "v1",
+			"metadata": {
+				"name": "e2e-test-configmap",
+				"namespace": "default"
+			},
+			"data": {
+				"keys": "image.public.key=771 \nrsa.public.key=42"
+			}
+		}`, nil)
+		Expect(err).NotTo(HaveOccurred())
+		cm, err := Clients.Kubernetes.CoreV1().ConfigMaps("default").Get("e2e-test-configmap", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		Expect(cm.Namespace).To(BeEquivalentTo("default"))
+		Expect(cm.Name).To(BeEquivalentTo("e2e-test-configmap"))
+
+		// Delete object and confirm via kubeclient
+		_, err = adminRequest("DELETE", "kubernetesobjects", "", nil, "kind=configmap", "namespace=default", "name=e2e-test-configmap")
+		Expect(err).NotTo(HaveOccurred())
+		cm, err = Clients.Kubernetes.CoreV1().ConfigMaps("default").Get("e2e-test-configmap", metav1.GetOptions{})
+		Expect(err).To(HaveOccurred())
```

Comment:
[nice to have] the error should be NotFound 
import "k8s.io/apimachinery/pkg/api/errors"

```suggestion
		Expect(errors.IsNotFound(err)).To(Equal(true))
```